modules/op-resolver-git-config-scm.adoc

```diff
@@ -0,0 +1,66 @@
+// This module is included in the following assembly:
+//
+// // *openshift_pipelines/remote-pipelines-tasks-resolvers.adoc
+
+:_content-type: PROCEDURE
+[id="resolver-git-config-scm_{context}"]
+= Configuring the Git resolver for the authenticated SCM API
+
+For the authenticated SCM API, you must set the configuration for the authenticatred Git connection.
+
+You can use Git repository providers that are supported by the link:https://github.com/jenkins-x/go-scm/tree/main/scm/driver[`go-scm` library]. Not all `go-scm` implementations have been tested with the Git resolver, but the following providers are known to work:
+
+* github.com and GitHub Enterprise
+* gitlab.com and self-hosted Gitlab
+* Gitea
+* BitBucket Server
+* BitBucket Cloud
+
+[NOTE]
+====
+You can configure only one Git connection using the authenticated SCM API for your cluster. This connection becomes available to all users of the cluster. All users of the cluster can access the repository using the security token that you configure for the connection.
+====
+
+[NOTE]
+====
+If you configure the Git resolver to use the authenticated SCM API, you can also use anonymous Git clone references to retrieve pipelines and tasks.
+====
+
+
+.Procedure
+
+. To edit the `TektonConfig` custom resource, enter the following command:
++
+[source,terminal]
+----
+oc edit TektonConfig config
+----
++
+In the `TektonConfig` custom resource, in the `pipeline.git-resolver-config` spec, specify  configuration:
++
+[source,yaml]
+----
+apiVersion: operator.tekton.dev/v1alpha1
+kind: TektonConfig
+metadata:
+  name: config
+spec:
+  pipeline:
+    git-resolver-config:
+      default-revision: main # <1>
+      fetch-timeout: 1m # <2>
+      scm-type: github # <3>
+      server-url: api.internal-github.com # <4>
+      api-token-secret-name: github-auth-secret # <5>
+      api-token-secret-key: github-auth-key # <6>
+      api-token-secret-namespace: github-auth-namespace # <7>
+      default-org: tektoncd # <8>
+----
+<1> The default Git revision to use if none is specified.
+<2> The maximum time any single Git clone resolution may take, for example, `1m`, `2s`, `700ms`. {pipelines-title} also enforces a global maximum timeout of 1 minute on all resolution requests.
+<3> The SCM provider type. Required for using the authenticated SCM API.
+<4> The base URL for use with the authenticated SCM API. Not required if using `\github.com`, `\gitlab.com`, or BitBucket Cloud.
```

Comment:
The "\" is rendering in front of github.com and gitlab.com. Were you trying to escape here? (not necessary if they're in backticks)